powerline/selectors/tmux.py

```diff
@@ -0,0 +1,34 @@
+# vim:fileencoding=utf-8:noet
```

Comment:
And I would actually suggest moving that to `powerline/selectors/proc.py` and get rid of any references to tmux here. Neither idea of the process tree traversal nor current implementation of the selector is really tmux-specific.

Reply:
The idea is good, however this requires the exclude function to be named `powerline.selectors.proc.is_ssh`.
I would propose to change this into a file named common.py and add this module to the general search path for the selectors